src/components/nav-bar/nav-bar.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { StyledBold, Logo, Navigation, StyledNav, StyledUl, StyledLi, Container} from "./nav-bar-styles";
+import { HashLink as Link } from 'react-router-hash-link';
+
+
+const NavBar = () => {
+    return (
+    <Navigation>
+    <Container>
+        <Logo>
+       <StyledBold>Paint</StyledBold>Utah<StyledBold>Curbs</StyledBold>
+        </Logo>
+        <StyledNav>
+            <StyledUl>
+         
+            <Link to="/#welcom"><StyledLi>Home</StyledLi></Link>
```

Comment:
welcome is spelled wrong here. Also do you want to add the smooth attribute here? i feel it is nice.